src/base/bittorrent/trackerentry.cpp

```diff
@@ -28,6 +28,33 @@
 
 #include "trackerentry.h"
 
+#include <QList>
+#include <QVector>
+
+QVector<BitTorrent::TrackerEntry> BitTorrent::parseTrackerEntries(const QStringView view)
```

Comment:
Dislike `view` as name for this parameter.

Reply:
Even if you don't want to bother with it, `str` would be more appropriate one. Using "string view" is just implementation detail in this case.

Reply:
Renamed.